build/cloud/ha/README.md

```diff
@@ -158,3 +158,33 @@ fi
 
 ```
 
+## haproxy
```

Comment:
I think the haproxy need to config with keeppalived together?

Reply:
> I think the haproxy need to config with keeppalived together?

Yes, I can sort out the configuration information of keepalived.